### DISCUSSION TITLE
ytsr & ytpl

### DISCUSSION THREAD

**Discussion by aka-pal** [eritislami/evobot#505] [Ideas]
I think we can use scarp instead of YouTube Api key. We can use ytsr and ytpl to search songs on YouTube anonymously. Also it will resolve yhe issue of not getting song information on playlist. We don't have to worry about YouTube Api token quote end.

**Comment by Perdition0** [comment]
How do I use ytsr and ytpl @L0SER8228

**Comment by wolf-yuan-6115** [comment]
`ytpl` and `ytsr` is unstable because YouTube will update website sometime, it's better to use YouTube API key

**Reply by aka-pal** [reply]
So, can you use ytsr & ytpl along with YouTube Api key
First it will try to search with ytsr ,if any error then try to search with YouTube Api key. Thus we can save Youtube api key usage qouta

**Reply by wolf-yuan-6115** [reply]
> So, can you use ytsr & ytpl along with YouTube Api key
> First it will try to search with ytsr ,if any error then try to search with YouTube Api key. Thus we can save Youtube api key usage qouta

Why don't just use YouTube API...
There will have more chance to get 429 error.
